github/repos_contents.go

```diff
@@ -119,6 +119,11 @@ func (s *RepositoriesService) DownloadContents(owner, repo, filepath string, opt
 	return nil, fmt.Errorf("No file named %s found in %s", filename, dir)
 }
 
+func escapeUrlPath(path string) string {
```

Comment:
Minor, `golint` would say this needs to be named `escapeURLPath`, since Go style uses MixedCaps style.